rebar.config

```diff
@@ -61,6 +61,12 @@
   {short, [{erl_opts, [{d, 'EQC_SHORT_TEST'}]}]},
   {long, [{erl_opts, [{d, 'EQC_LONG_TEST'}]}]},
   {prod, [{relx, [{dev_mode, false}]}]},
+  {deb, 
+	  [{relx, 
+		  [{dev_mode, false},
+		  {overlay_vars, "rel/vars/deb.config"}]},
+	  {cuttlefish,
```

Comment:
sure, I don't mind changing it - the current default is `dalmatinerdb.conf.example`, defined here: https://github.com/dalmatinerdb/dalmatinerdb/blob/dev/rebar.config#L40